### ISSUE TITLE
Assessing consensus in the CG Charter

### ISSUE THREAD

**Issue by evanp** [swicg/potential-charters#30]
The charter boilerplate says this:

> This group will seek to make decisions where there is consensus. Groups are free to decide how to make decisions (e.g. Participants who have earned Committer status for a history of useful contributions assess consensus, or the Chair assesses consensus, or where consensus isn't clear there is a Call for Consensus [CfC] to allow multi-day online feedback for a proposed course of action). It is expected that participants can earn Committer status through a history of valuable contributions as is common in open source projects. After discussion and due consideration of different opinions, a decision should be publicly recorded (where GitHub is used as the resolution of an Issue).

It doesn't seem like we can just leave this open. I think we could maybe follow a process like this:

- Chair(s) assess consensus
- Chair(s) can delegate consensus assessment to leaders of a task force (which seems like it'd be a big overlap with "Committers" used here)
- Chair(s) or task force leads use CfC when consensus isn't clear.

I think the "straw poll" mechanism we've used in the past is one way to assess consensus; there are probably others.

**Comment by gobengo** [comment]
tl;dr: the charter text currently allows for the process you suggest. can we do nothing?

> It doesn't seem like we can just leave this open.

Why doesn't it seem that way?

> I think the "straw poll" mechanism we've used in the past is one way to assess consensus; there are probably others.

It assesses consensus amongst participants in a meeting, but not participants in the whole group, nor other stakeholders that W3C groups must reach consensus with.

There are definitely other ways to assess consensus and document its evidence, including the CFC that's in the charter template and many other CG charters.
The CFC on-list + time to object is, in my view, necessary to provide evidence of groupwide consensus including a diverse pool of participants across many time zones i.e. not in one single meeting. The asynchronous collaboration time after a sync straw poll also encourages good decisions. For more on the benefits of async collaboration see https://www.asyncagile.org/perspectives

I see in your hypothetical process the bit about delegating assessment to task force leaders. I think this is already implied by the existing text saying "the Chair assesses consensus". IMHO it's best for the charter to neither endorse this delegation nor forbid it. i.e. the charter text currently allows for the process your suggest.

**Comment by evanp** [comment]
The text says:

> Groups are free to decide how to make decisions (e.g. Participants who have earned Committer status for a history of useful contributions assess consensus, or the Chair assesses consensus, or where consensus isn't clear there is a Call for Consensus [CfC] to allow multi-day online feedback for a proposed course of action).

That sounds to me like we're supposed to decide, even if the decision is "any or all of these".

I like the idea of devolving this responsibility to taskforce leads (I think that's about the same as "Committee") where possible, and letting chairs do it for group-wide decisions.

**Comment by evanp** [comment]
If we're on the same page, on the process, and just differing on the wording, would you be open to text like this?

> The Group makes decisions in the following ways: either Participants who have earned Committer status for a history of useful contributions assess consensus, or the Chair assesses consensus, or where consensus isn't clear there is a Call for Consensus [CfC] to allow multi-day online feedback for a proposed course of action.

That sounds less, to me, like there is an open question we need to resolve.
